src/deepsparse/transformers/haystack/__init__.py

```diff
@@ -13,8 +11,130 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Helpers for running haystack pipelines and nodes with DeepSparse and integrating
+deepset-ai/haystack
+"""
 
 # flake8: noqa
+
+import logging as _logging
+
+
+_HAYSTACK_PREFERRED_VERSION = "1.4.0"
+_HAYSTACK_EXTRAS = "[all]"
+
+# trigger transformers download
+from deepsparse import transformers as _transformers
+
+
+# check haystack installation
+try:
+    import haystack as _haystack
+
+    if _haystack.__version__ != _HAYSTACK_PREFERRED_VERSION:
+        raise ValueError(
+            f"Deepsparse requires farm-haystack=={_HAYSTACK_PREFERRED_VERSION}, "
+            f"but found {_haystack.__version__}"
+        )
+    _haystack_import_error = None
+except Exception as _haystack_import_err:
+    _haystack_import_error = _haystack_import_err
+
+_LOGGER = _logging.getLogger(__name__)
+
+# used only for error messages
+_NM_TRANSFORMERS_TAR_TEMPLATE = (
```

Comment:
There shouldn't be a need to install transformers since the parent directory takes care of that